.circleci/test-deploy.yml

```diff
@@ -0,0 +1,55 @@
+version: 2.1
+orbs:
+  gradle: entur/gradle-orb@dev:<<pipeline.git.revision>>
+  orb-tools: circleci/orb-tools@11.6.1
+filters: &filters
+  tags:
+    only: /.*/
+jobs:
+  # This job is an example of an integration testing job.
+  # This job should execute a command from your orb and verify
+  # the output is as expected, otherwise the job should fail.
+  #
+  # Rename this job and add more as needed.
+  #
+  integration-test-1:
```

Comment:
This test doesn't do much, it's more of an expectation test that the build image has gradle installed. It's debatable if this is necessary for the `gradle/default` executor.